dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageCheckOption.java

```diff
@@ -89,53 +80,66 @@ private static void checkInput(CommandLine commandLine) throws ParseException {
 
   private static Options configureOptions() {
     Options options = new Options();
+
     options.addOption(
         "b", "bom", true, "BOM to generate a class path, specified by its Maven coordinates");
-    options.addOption(
-        "a",
-        "artifacts",
-        true,
-        "Maven coordinates for Maven artifacts (separated by ',') to generate a class path");
-    options.addOption("j", "jars", true, "Jar files (separated by ',') to generate a class path");
+
+    Option artifactOption = Option.builder("a").longOpt("artifacts").hasArgs()
+        .valueSeparator(',')
+        .desc("Maven coordinates for Maven artifacts (separated by ',') to generate a class path")
+        .build();
+    options.addOption(artifactOption);
+
+    Option jarOption = Option.builder("j").longOpt("jars").hasArgs()
+        .valueSeparator(',')
+        .desc("Jar files (separated by ',') to generate a class path")
+        .build();
+    options.addOption(jarOption);
+
     options.addOption(
         "r",
         "report-only-reachable",
         false,
         "To report only linkage errors reachable from entry point");
-    options.addOption(
-        "m",
-        "maven-repository",
-        true,
-        "Maven repository URL to search for dependencies. "
-            + "If this option is used one or more times, the repositories are added "
-            + "in order before the default Maven Central (http://repo1.maven.org/maven2/).");
+
+    Option repositoryOption = Option.builder("m").longOpt("maven-repositories").hasArgs()
+        .valueSeparator(',')
+        .desc("Maven repository URLs to search for dependencies. "
+            + "The repositories are added to a repository list in order before "
+            + "the default Maven Central (http://repo1.maven.org/maven2/).")
+        .build();
+    options.addOption(repositoryOption);
+
+    Option noMavenCentralOption = Option.builder("nm").longOpt("no-maven-central")
+        .hasArg(false)
+        .desc("Used with '-m'. Not to add Maven Central to the repository list. "
```

Comment:
Instead of "Used with '-m'. Not to add Maven Central to the repository list.", let's say "Do not search Maven Central in addition to the repositories specified by -m."

Reply:
Updated.